src/coreComponents/unitTests/meshTests/testVTKImport.cpp

```diff
@@ -56,7 +56,7 @@ void TestMeshImport( string const & meshFilePath, V const & validate )
 
 TEST( VTKImport, cube )
 {
-  auto validate = []( CellBlockManagerABC const & cellBlockManager ) -> void
+  auto validate = []( CellBlockManagerABC & cellBlockManager ) -> void
```

Comment:
BTW, since `CellBlockManagerABC` still has its `const` member functions, you surely can put the `const` back here?